src/Avalonia.Base/Data/Core/Parsers/ExpressionTreeParser.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Text;
+
+namespace Avalonia.Data.Core.Parsers
+{
+    class ExpressionTreeParser
+    {
+        private readonly bool enableDataValidation;
```

Comment:
This applies to quite a few places, but elsewhere we use `_field` with an underscore for fields to follow the .net core coding guidelines.